documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -69,30 +108,81 @@ value.converter.apicurio.registry.url=http://apicurio:8080/api
+.. Deploy the latest published {registry} operator by running the following command:
++
+`curl -sSL https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/install.yaml | sed "s/{NAMESPACE}/$NAMESPACE/g" | oc apply -f -`
++
+This deploys the latest development version of the {registry} operator from the `master` branch. To deploy other versions, specify a different branch or tag, or edit the operator image reference in the file.
++
+.. Create a new {registry} deployment by specifying the in-memory persistence option in one of the example custom resources, for example:
++
+`oc create -f https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/example-cr/in-memory.yaml`
++
+The in-memory deployment is not suitable for production. Use the Kafka Streams persistence option for production. For more information, see {LinkServiceRegistryGetStart}[NameServiceRegistryGetStart].
+endif::product[]
+
+. Build a {prodname} container image that implements the Avro converter:
++
+ifdef::community[]
+.. Copy link:https://github.com/debezium/debezium-examples/blob/master/tutorial/debezium-with-apicurio/Dockerfile[`Dockerfile`] to a convenient location. This file has the following content: 
++
+[listing,options="nowrap"]
+----
+ARG DEBEZIUM_VERSION
+FROM debezium/connect:$DEBEZIUM_VERSION
+ENV KAFKA_CONNECT_DEBEZIUM_DIR=$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-mysql
+ENV APICURIO_VERSION=1.1.2.Final
 
-Build a {prodname} image with Avro converter from https://github.com/debezium/debezium-examples/blob/master/tutorial/debezium-with-apicurio/Dockerfile[Dockerfile]:
+RUN cd $KAFKA_CONNECT_DEBEZIUM_DIR &&\
+    curl https://repo1.maven.org/maven2/io/apicurio/apicurio-registry-distro-connect-converter/$APICURIO_VERSION/apicurio-registry-distro-connect-converter-$APICURIO_VERSION-converter.tar.gz | tar xzv
+----
 
-[source]
-[subs="attributes"]
+.. Run the following command: 
++
+[source,subs="attributes"]
 ----
 docker build --build-arg DEBEZIUM_VERSION={debezium-docker-label} -t debezium/connect-apicurio:{debezium-docker-label} .
 ----
-
-Run a Kafka Connect image configured to use Avro:
-
-[source]
-[subs="attributes"]
+endif::community[]
+ifdef::product[]
+.. Download the link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=red.hat.integration&downloadType=distributions[{registry} Kafka Connect] zip file. 
+.. Extract the content into the directory that contains the {prodname} connector that you are configuring to use Avro serialization. 
+.. Create a custom image for Kafka Connect. See link:{LinkCDCInstallOpenShift}[NameCDCInstallOpenShift] for an example of how to do this. Start with the `Dockerfile` in that example. Then add the {registry} converters to the connector directories. 
+endif::product[]
+
+. Run a Kafka Connect image that is configured to use Avro:
```

Comment:
```suggestion
. Run the newly built Kafka Connect image, configuring it so it uses the Avro converter (or configure specific converters to do so):
```

Reply:
I don't understand the last part of your suggestion. 
What do you mean by "configure specific converters to do so"? -- It sounds like this says that you can configure other converters to use the Avro converter. Is that what you mean?

Reply:
Ah, sorry, I meant "specific connectors". Converters can be configured globally for the entire Connect worker or individually for connectors. As commented somewhere else, I think ideally we'd only discuss the latter.

Reply:
Fixed for now. Will revisit to do better.